enable_optimizer: true
enable_actions_concat: true
enable_spline: true
enable_temporal_ensemble: false
temporal_ensemble_coeff: 0.9
action_dim: 7
chunk_size: 100
n_action_step: 100 # 100
blending_horizon: 10 # 10
len_delay_time: 5 # 5
spline: "linear" # "quintic" or "cubic" or "linear"
optimizer_solver: "CLARABEL"
policy_dir: "towel_flattenings_777_model"
dataset_dir: "towel_flattening"
enable_actions_publish: true
